pkg/cli/register.go

```diff
@@ -43,8 +46,24 @@ func Execute() {
 	cobra.OnInitialize(func() {
 		// Set up the logger
 		logging.Init(LogType, LogLevel)
+
+		var configfile string
+		if len(ConfigFile) > 0 {
+			configfile = ConfigFile
+		}
+
+		if len(configfile) == 0 {
+			configfile = os.Getenv(configEnvvarName)
+		}
+
+		zap.S().Debugf("%s=%s", configEnvvarName, os.Getenv(configEnvvarName))
```

Comment:
if env variable is not specified, we would log:
`TERRASCAN_CONFIG=`, is that okay?
also, we are using `:` when logging a value.

Reply:
changes made